android/app/src/main/res/values/styles.xml

```diff
@@ -1,9 +1,9 @@
 <resources>
-
     <!-- Base application theme. -->
     <style name="AppTheme" parent="Theme.AppCompat.DayNight.NoActionBar">
         <!-- Customize your theme here. -->
         <item name="android:editTextBackground">@drawable/rn_edit_text_material</item>
+        <item name="android:textColor">#000</item>
```

Comment:
What is this used for?

Reply:
Actually in android the texts and icons are being render in grey color and to overcome this i have added in styles.xml. After adding this line the texts and icons are now being showed in black color.

Reply:
OKay but we should always set color to text in react native side. Showing grey color could be an indication that color was not set in RN